client/js/game.js

```diff
@@ -4667,15 +4667,15 @@ function Game() {
             this.soundBox(ctx,r1,c3+13,"soundclick","sound","click","Click",data.click,"Mutes Click Sound");
 
             this.checkBox(ctx,r2-20,c2-80,data.showNumbers,"sshown","toggle","showNumbers","Show clicks","right","Display the energy obtained by Clicks");
-            this.checkBox(ctx,r2-20,c2-20,data.fractal,"sfrac","toggle","fractal","High Graphics","right","Disable for better performance");
-            this.checkBox(ctx,r2-20,c2+40,data.tdon,"stdon","toggle","tdon","Energy Orbs","right","Run/Pause Energy Orbs");
+            this.checkBox(ctx,r2-20,c2-35,data.fractal,"sfrac","toggle","fractal","High Graphics","right","Disable for better performance");
+            this.checkBox(ctx,r2-20,c2+10,data.tdon,"stdon","toggle","tdon","Energy Orbs","right","Run/Pause Energy Orbs");
+            this.checkBox(ctx,r2-20,c2+55,data.colorblind,"scolorblind","toggle","colorblind","Color Blind","right","Display Numbers in AP board");
             
             if (CQW!==undefined && CQW.isPublic!==undefined) this.checkBox(ctx,r2-20,c2+100,CQW.isPublic,"spub","toggle","public","Display name","right","Show your name in CosmosQuest.net");
 
-            text(ctx,"Notation",r3+T.width("0567")/2,c1-ndist,"30px "+FONT,"white","center","top");
-            this.rectButton(ctx,r3,c1,"notation","notation",undefined,undefined,["S.I.","Scientific"],data.nMode,"Changes Number Display Mode");
-            text(ctx,"Color Blind",r4+T.width("0567")/2,c1-ndist,"30px "+FONT,"white","center","top");
-            this.rectButton(ctx,r4,c1,"colorblind","cblind",undefined,undefined,["OFF","ON"],data.colorblind,"Display Number in AP board");
+            text(ctx,"Notation",r3+T.width("0567")+5,c1-ndist,"30px "+FONT,"white","center","top");
+            this.rectButton(ctx,r3,c1,"notation","notation",undefined,undefined,["S.I.","Scientific"],data.nMode,"Changes Number Display Mode for Idle");
+            this.rectButton(ctx,r4,c1,"bintmode","bintmode",undefined,undefined,["English","S.I."],data.bintmode,"Changes Number Display Mode for Followers");
```

Comment:
I didn't come up with a better short description than "English" for k,M,B,T,Q...

Reply:
For clarity, can we have 3 options ?
- k/M/G/T/P
- k/M/B/T/Q
- k/Mn/Md/Bn/Bd

Reply:
SHouldn't be a propblem, how would you call that third mode?

Reply:
What about calling them "k/M/B/T/Q" etc ? Otherwise the only words I can find are "Short" and "Long", and many people won't get it...

Reply:
![grafik](https://user-images.githubusercontent.com/24701475/91994544-002aa380-ed37-11ea-97bd-704a0f1366d1.png)
(would be too long otherwise)

Reply:
Good enough for me. Let's do that and we'll see if Gaia agrees.